src/lib/viewers/BaseViewer.js

```diff
@@ -666,8 +668,10 @@ class BaseViewer extends EventEmitter {
 
         const { file } = this.options;
         this.canAnnotate = checkPermission(file, PERMISSION_ANNOTATE);
+        this.canViewAllAnnotations = checkPermission(file, PERMISSION_CAN_VIEW_ANNOTATIONS_ALL);
+        this.canViewOwnAnnotations = checkPermission(file, PERMISSION_CAN_VIEW_ANNOTATIONS_SELF);
 
-        if (this.canAnnotate) {
+        if (this.canAnnotate || this.canViewAllAnnotations || this.canViewOwnAnnotations) {
```

Comment:
Could we abstract this into the annotator util we have? Then you could do something like:
```
if(annotatorUtil.enabledForFile(file)) {
  this.initAnnotations();
}
```

Reply:
Moved the permissions check into BoxAnnotations.determineAnnotator() so that we don't introduce more annotations specific code into BaseViewer.js